src/Arcus.POC.Observability.Telemetry.Serilog.Sinks.ApplicationInsights/Extensions/ILoggerExtensions.cs

```diff
@@ -240,6 +242,7 @@ public static class ILoggerExtensions
             this ILogger logger,
```

Comment:
I think these extended HTTP request tracking extensions are now not needed anymore if we use the updated observability package.

Reply:
I think we can have a look at that in a separate PR ?

Reply:
We could, it's just that this PR contains both package updates as usage of new features, so it seemed related to the changes already made. But sure, no problem.

Reply:
It would be too much to do it in this PR.